upgrade.php

```diff
@@ -1633,6 +1633,20 @@ function upgrade_1835_mysql() {
     db_query_parsed("ALTER TABLE `$table` CHANGE `timestamp` `timestamp` {DATETIME}");
 }
 
+function upgrade_1836_mysql() {
+    # alternative contact means to reset a forgotten password
+
+    foreach(array('admin', 'mailbox') as $table_to_change) {
+        $table = table_by_key($table_to_change);
+        if(!_mysql_field_exists($table, 'phone')) {
+            db_query_parsed("ALTER TABLE `$table` ADD COLUMN `phone` varchar(30) NOT NULL DEFAULT ''");
+        }
+        if(!_mysql_field_exists($table, 'email_other')) {
+            db_query_parsed("ALTER TABLE `$table` ADD COLUMN `email_other` varchar(255) NOT NULL DEFAULT ''");
+        }
+    }
+}
+
 # TODO MySQL:
```

Comment:
please use the _db_add_field() function instead of doing manual SQL queries. Also, please specify the encoding (unsing `{LATIN1}` or `{UTF-8}`) for all text fields you add. (See db_query_parsed() for a list of all suported `{...}` markers.)